lib/replace-in-file.js

```diff
@@ -84,5 +84,9 @@ replaceInFile.sync = function(config) {
   return paths.map(path => replaceSync(path, from, to, config));
 };
 
+replaceInFile.replaceInFile = replaceInFile; // Self-reference to support named import
+replaceInFile.replaceInFileSync = replaceInFileSync;
+replaceInFile.sync = replaceInFileSync;
```

Comment:
What's the benefit of having both `sync` and `replaceInFileSync`?

Reply:
`sync` is just a legacy alias for backward compatibility
new named import let to avoid default module reference
```typescript
// Before
import replaceInFile, {sync as replaceInFileSync} from 'replace-in-file'
// After
import {replaceInFile, replaceInFileSync} from 'replace-in-file'
```